designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/FullScreenImage.kt

```diff
@@ -81,9 +71,10 @@ fun FullScreenImage(
         ),
         onDismissRequest = onDismiss,
     ) {
+        opened = true
```

Comment:
Use `LaunchedEffect(Unit)` to change this? or else it would get triggered for every recomposition.